content/en/blog/_posts/2022-11-28-advancements-in-traffic-engineering/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: blog
+title: "Kubernetes v1.26: Advancements in Kubernetes Traffic Engineering"
+date: 2022-11-28
```

Comment:
```suggestion
date: 2022-12-30
```
Please also change the path `content/en/blog/_posts/2022-11-28-advancements-in-traffic-engineering/index.md` to match.

Reply:
Updated publication date, thanks